### PR TITLE
FCREPO-2093 (Remove special case for RDF binary resources), applied to 4.6.0-RC

### PR DESCRIPTION
**Pull request by birkland**

### CODE REVIEW

File: fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java [hunk 1]
```diff
@@ -167,41 +167,19 @@ protected Response getContent(final String rangeValue,
...
-                    } catch (final URISyntaxException e) {
-                        throw new RepositoryRuntimeException(e);
-                    }
+            final MediaType mediaType = MediaType.valueOf(((FedoraBinary) resource()).getMimeType());
+
+            if (MESSAGE_EXTERNAL_BODY.isCompatible(mediaType) && mediaType.getParameters().containsKey(
+                    "access-type") && mediaType.getParameters().get("access-type").equals("URL") && mediaType
+                    .getParameters().containsKey("URL")) {
+                try {
+                    return temporaryRedirect(new URI(mediaType.getParameters().get("URL"))).build();
```
Comment (ajs6f) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java]: If you use `URI.create` here, can we avoid the `try-catch`?

Comment (birkland) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java]: Works for me, build passes

File: fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java [hunk 2]
```diff
@@ -980,6 +981,30 @@ public void testIngestWithBinary() throws IOException {
...
+        method.setEntity(new StringEntity(rdf));
+
+        try (final CloseableHttpResponse response = execute(method)) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+
+            final String location = getLocation(response);
+            final HttpGet get = new HttpGet(location);
+
+            try (final CloseableHttpResponse getResponse = execute(get)) {
+                final String resp = IOUtils.toString(getResponse.getEntity().getContent());
```
Comment (ajs6f) [inline fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java]: Prefer `EntityUtils.toString` which is more specific and produces fewer short-lived objects. Although at this point I'm being really picky.

Comment (acoburn) [inline fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java]: This is just the test suite, after all.

Comment (ajs6f) [inline fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java]: Yeah, I'm kind of a jerk. Feel free to merge this, I'm not near a workstation.

**Pull request merged**